### DISCUSSION TITLE
[Suggestion/Request] Chat log save location.

### DISCUSSION THREAD

**Discussion by whitepawrolls** [cinderblocks/radegast#18] [Ideas]
Would love to see a way to save chat logs in a designated location. I have my logs on Firestorm sent to a custom folder for example which then gets backed up to the cloud. I can't do that with radegast, so can't use it near as often as I want to.

**Comment by cinderblocks** [comment]
Excellent idea

**Reply by whitepawrolls** [reply]
Thank you for getting a location for storing the logs in, but I would like to suggest one change. As it is now it just puts all logs into one folder. If you have multiple accounts you log into its going to get them mixed up. My suggestion is to do like LL/Firestorm does and automatically create a sub folder inside the selected location with the name of the logged in account and store each accounts logs separately like this. Thank you again

**Reply by cinderblocks** [reply]
That was completely by oversight. Let me correct it at once!

**Reply by whitepawrolls** [reply]
Thank you again for the fast response. Is there a total noob's guide on how to compile this so I don't have to wait for the next release version for the fix? Thank you again :)

**Reply by cinderblocks** [reply]
It is pretty easy to do. If you are using Windows, you simply need to obtain the latest version of Visual Studio (I believe Community Edition is free) and open Radegast.sln with it. Then change the build type to Release, then click Build. It will output the final package.

If you don't want to go through the trouble, a development version is here: https://ci.appveyor.com/api/buildjobs/w6t9lrk7k8hrn52s/artifacts/RadegastSetup_x64-3.0.348.exe

**Reply by meganlawalert** [reply]
Don't forget to install WIX tools after installing Visual Studio Community.  They are free and simple to add, but needed to build the Radegast solution.

**Reply by whitepawrolls** [reply]
Thank you, but I gave up on this months ago. I much prefer compiling things in linux to installing all this stuff to do it in windows, and having to learn it. I'll leave that to the pros, and just get the releases as they are available. Speaking of in the "check for updates" part of the menu it needs updating as its still listing the old update and failing. Thank you again.
